modules/nf-core/biohansel/main.nf

```diff
@@ -0,0 +1,45 @@
+process BIOHANSEL {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::bio_hansel=2.6.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bio_hansel:2.6.1--py_0':
+        'quay.io/biocontainers/bio_hansel:2.6.1--py_0' }"
+
+    input:
+    tuple val(meta), path(seqs)
+    val scheme
```

Comment:
You can add this to your meta and supply this in the `args` similar to this [example](https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/gatk4/printsvevidence/nextflow.config). Input fields should be reserved for actual file inputs and all values should be in `meta` according to the `nf-core` guidelines on modules

Reply:
Or you could just supply the raw value in `modules.config` (that would be even easier if you always use the same scheme)